quarkus/pom.xml

```diff
@@ -169,5 +169,105 @@
             <url>https://repository.jboss.org/nexus/content/groups/public/</url>
         </repository>
     </repositories>
-    
+
+    <profiles>
+        <profile>
+            <!--
+                This profile is intended for building the distribution.
+                It should be enabled whenever building the distribution so that only the expected
+                artifacts are included within the distribution.
+                By
+            -->
+            <id>distribution</id>
+            <dependencyManagement>
+                <dependencies>
```

Comment:
I'm not really following this - why would there be a "distribution" profile, and not just remove the deps always?

Reply:
When running tests we don't want to completely disable the test capabilities from quarkus such as dev services / testcontainers.

Reply:
In that case wouldn't you just use the test scope for the dependency instead of a profile?